deploy/install-phantomjs.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
```

Comment:
`set -e` here. We want to make sure the docker build operation fails in CI if this script fails.